src/vma/util/agent.h

```diff
@@ -35,65 +35,64 @@
 
 #include "vma/util/agent_def.h"
 
+/**
+ * @struct agent_msg_t
+ * @brief Agent message resource descriptor.
+ *
+ * This structure describes a internal message object.
+ */
 typedef struct agent_msg {
-	struct list_head item;
-	int length;
+	struct list_head item;             /**< link element */
+	int length;                        /**< actual length of valuable data */
+	intptr_t tag;                      /**< unique identifier of the message */
```

Comment:
do we need the tag field ? i dont see a place where we use it

Reply:
I would like to keep this field as far as this protocol was designed to be flexible (not specific for TCP state info update).